src/main/java/frc/robot/Robot.java

```diff
@@ -44,7 +64,49 @@
   public void robotInit() {
     m_oi = new OI();
     // chooser.addOption("My Auto", new MyAutoCommand());
-    SmartDashboard.putData("Auto mode", m_chooser);
+    //SmartDashboard.putData("Auto mode", m_chooser);
+
+    /*
+    // hatchCamera = CameraServer.getInstance().startAutomaticCapture("hatch",0);
+    // hatchCamera.setResolution(640, 480);
+     portCamera = CameraServer.getInstance().startAutomaticCapture("port",1);
+     portCamera.setResolution(1280, 960);
+
+    new Thread(()->{
+      UsbCamera camera = CameraServer.getInstance().startAutomaticCapture(0);
+      camera.setResolution(640, 480);
+      CvSink cvSink = CameraServer.getInstance().getVideo();
+      CvSource outputStream = CameraServer.getInstance().putVideo("7042", 640, 480);
+      Mat source = new Mat();
+      Mat output = new Mat();
+      while(!Thread.interrupted()){
+        cvSink.grabFrame(source);
+        Imgproc.cvtColor(source, output, Imgproc.COLOR_BGR2GRAY);
+        outputStream.putFrame(output);
+      }
+    }).start();
+    */
+		Cameras.setup(); // Setup and Connection to Pixy2 and Microsoft Camera
+		SmartDashboard.putNumber("Iteration", myCounter);
+    try {
+			/***********************************************************************
+			 * navX-MXP:
+			 * - Communication via RoboRIO MXP (SPI, I2C, TTL UART) and USB.            
+			 * - See http://navx-mxp.kauailabs.com/guidance/selecting-an-interface.
+			 * 
+			 * navX-Micro:
+			 * - Communication via I2C (RoboRIO MXP or Onboard) and USB.
+			 * - See http://navx-micro.kauailabs.com/guidance/selecting-an-interface.
+			 * 
+			 * Multiple navX-model devices on a single robot are supported.
+			 ************************************************************************/
+            //ahrs = new AHRS(SerialPort.Port.kUSB1);
+            ahrs = new AHRS(SerialPort.Port.kMXP, SerialDataType.kProcessedData, (byte)50);
```

Comment:
Is this line causing runtime errors?  If you try-catch this and it fails, the code will not try again in robotInit.  This error may occur if the navX is not installed or working properly, I presume